.bazelci/presubmit.yml

```diff
@@ -581,6 +581,8 @@ tasks:
       - "--android_platforms=//:arm64-v8a"
     build_targets:
       - "//:android_app"
+    environment:
+      ANDROID_NDK_HOME: /opt/android-ndk-r25b
```

Comment:
Can you leave a comment on how this value was found in case it needs to be updated in the future?

Reply:
Good catch. Done.